packages/mysql/test/unit/mysql_p.test.js

```diff
@@ -141,7 +141,7 @@ describe('captureMySQL', function() {
         var stubDataInit = sandbox.stub(SqlData.prototype, 'init');
         var config = conn.config;
 
-        query.call(connectionObj, 'sql here');
+        query.call(connectionObj, 'sql here', [1]);
```

Comment:
Just trying to get a better understanding of this, do you mind clarifying the purpose of passing in `[1]` here?